src/pages/coaches/david-leuliette.md

```diff
@@ -0,0 +1,33 @@
+---
+name: David Leuliette
+title: Senior Front-end Engineer
+companyName: Empire fast
+companyLink: https://davidl.fr
+gitHub: https://github.com/flexbox
+medium: https://medium.com/@flexbox
+twitter: https://twitter.com/flexbox_
+linkedIn: https://uk.linkedin.com/in/david-leuliette
+blockquote: When you work 8 hours a day with a tool, every second matters.
+imageDescription: React GraphQL Academy coach David Leuliette, jumping on the Brooklyn Bridge
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Fdavid.jpg?alt=media
+tags: []
+---
+
+ <ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists